AndroidManifest.xml

```diff
@@ -16,7 +16,8 @@
         android:largeHeap="true"
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
-        android:label="@string/app_name" >
+        android:label="@string/app_name"
+        android:name="io.branch.referral.BranchApp">
```

Comment:
I love that we're relying on an Application class but I think for a majority of our partners, they'll already have one. I think we should create our own Application class (and perhaps have it retrieve the monster stored) and show init inside there?